app/templates/main/poc-browse.html

```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %}
```

Comment:
Havent reviewed this code as Colin's real browse page to supersede this

Reply:
the view will be superseeded with frontend browse view, so no need to review it